fixtures/conch-v3/datacenter.yaml

```diff
@@ -18,15 +18,15 @@ interactions:
       Cache-Control:
       - no-cache
       Date:
-      - Mon, 13 Jan 2020 23:16:20 GMT
+      - Sun, 26 Jan 2020 19:04:25 GMT
```

Comment:
This is just churn and committing these is surely not needed.

what are you doing to generate these changes? when I `make test`, nothing is altered in the repository.

Reply:
To generate these files simply remove `fixtures/conch-v3/*.yaml` and they'll be recreated on the next call to `go test` (which `make test` calls) by attempting to hit whatever Conch API endpoint is configured in your shell. I create them locally by hitting a fresh dev server with an empty database.

I've opened  #44 to track replacing this system with something less ugly. Sungo specifically warned me that Lane had tried something like this and it sucked, and it does … I'm just not sure how I want to replace everything. Half re-implementing the API using Gock isn't really more appealing … yet.

Reply:
Also the most recent rebase reduces the number of fixtures changed to just the ones directly impacted.